corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/utils_spec.js

```diff
@@ -45,7 +45,7 @@ hqDefine("cloudcare/js/form_entry/spec/utils_spec", function () {
             [groupInRepeat] = repeat.children();
             [textInRepeat] = groupInRepeat.children();
 
-            assert.equal(groupInRepeat.caption(), null);
+            assert.equal(groupInRepeat.caption, null);
```

Comment:
I'm not sure why this was affected this way. I could not see its connection with the change as this is related to repeat, which is not altered.

Reply:
is `groupInRepeat` pointing to the wrong element? i'm not sure. But could you track down what the cause, otherwise I think it's hiding a real bug. I don't think `groupInRepeat.caption` exists so its returning null and this assertion passes. For example, if you do `groupInRepeat.fakeprop`, it also asserts equal

Reply:
`groupInRepeat` was pointing to `grouped-element-tile-row`, so modified it to use its child, which is pointing to sub-group now that was initially created with `groupInRepeat`.

Reply:
resolved here https://github.com/dimagi/commcare-hq/pull/33834#discussion_r1430650776